charts/vault-instance/templates/pvc.yaml

```
@@ -4,9 +4,11 @@ metadata:
   name: vault-file
 spec:
   # https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1
-  # storageClassName: ""
+{{- if .Values.pvc.storageClassName }}
+  storageClassName: '{{ .Values.pvc.storageClassName }}'
+{{- end }}
```

The field `storageClassName` is now included only if the `.Values.pvc.storageClassName` is specified. This is breaking change. Previous behavior was forcing the usage of pre-provisioned volumes. Replace this with following to ensure that correct behavior is preserved:

```suggestion
  storageClassName: {{ .Values.pvc.storageClassName | default "" }}
```